test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -59,44 +57,12 @@ export class BridgePage {
     );
   };
 
-  navigateToAssetPage = async (
-    contractRegistry: GanacheContractAddressRegistry,
-    symbol: string,
-    shouldImportToken = true,
-  ) => {
-    if (shouldImportToken) {
-      // Import token
-      const contractAddress = await contractRegistry.getContractAddress(
-        SMART_CONTRACTS.HST,
-      );
-      await clickNestedButton(this.driver, 'Import tokens');
-      await clickNestedButton(this.driver, 'Custom token');
-      await this.driver.fill(
-        LOCATOR.MM_IMPORT_TOKENS_MODAL('custom-address'),
-        contractAddress,
-      );
-      await this.driver.fill(
-        LOCATOR.MM_IMPORT_TOKENS_MODAL('custom-symbol'),
-        symbol,
-      );
-      await this.driver.waitForSelector(
-        LOCATOR.MM_IMPORT_TOKENS_MODAL('custom-decimals'),
-      );
-      await clickNestedButton(this.driver, 'Next');
-      await this.driver.clickElement(
-        LOCATOR.MM_IMPORT_TOKENS_MODAL('import-button'),
-      );
-      await this.driver.assertElementNotPresent(
-        '[data-testid="import-tokens-modal-import-button"]',
-      );
-      await this.driver.clickElement({ text: symbol });
-    } else {
-      await this.driver.clickElement({
-        css: '[data-testid="multichain-token-list-button"]',
-        text: symbol,
```

Comment:
the test was flaky at this point, but now we remove all these steps as they are not needed